src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -245,6 +248,51 @@ def export_config(cmd,
         __write_kv_to_app_service(cmd, key_values=src_kvs, appservice_account=appservice_account)
 
 
+def export_secret(cmd,
+                  name=None,
+                  connection_string=None,
+                  label=None,
+                  key=None,
+                  prefix="",
+                  path=None,
+                  format_=None):
```

Comment:
>format_ [](start = 18, length = 7)

What is the default format?

Reply:
We don't have default format. If no --path specified, then this command will return secrets, as storing secrets in persist files may be a concern for customers. If --path specified but no --format, then we will just create an empty file. Same behavior like kv export.

Reply:
In that case, it may be better to call it "get-secret" instead of "export-secret". I feel like the expectation with "export" commands is to save the output somewhere. If customers only want to see their secret values in the terminal, or optionally save it in a file, "get" pattern may be better. And in that case, we wouldn't even have to support file formats, as users can pipe the output from CLI in any format they like.

---
In reply to: [398245409](https://github.com/Azure/azure-cli/pull/12737#discussion_r398245409) [](ancestors = 398245409)

Reply:
Or we could use "get-keyvault" since we already have a "set-keyvault" command.

---
In reply to: [398254227](https://github.com/Azure/azure-cli/pull/12737#discussion_r398254227) [](ancestors = 398254227,398245409)